pylint_odoo/checkers/modules_odoo.py

```diff
@@ -413,7 +413,8 @@ def _check_imported_packages(self, node, module_name):
 
         ext_deps = self.manifest_dict.get('external_dependencies') or {}
         py_ext_deps = ext_deps.get('python') or []
-        if module_name not in py_ext_deps and \
+        if not node.as_string().split()[1].startswith('.') and \
```

Comment:
You can use the same way from https://github.com/PyCQA/pylint/pull/888#r62025010

If is instance of `astroid.ImportFrom` class then you can use `node.level >= 1` in order to detect a `from . import package`

Reply:
@moylop260 
Thanks for your help, now I used the node.level to avoid the node.as_string